src/clj/rems/db/applications.clj

```diff
@@ -22,6 +22,11 @@
             [rems.workflow.dynamic :as dynamic])
   (:import [java.io ByteArrayOutputStream FileInputStream]))
 
+;; TODO could use schemas for these coercions
+(defn- fix-workflow-from-db [wf]
```

Comment:
I'd maybe keep this and the is-handler? function on the bottom of the file with the rest of the dynamic stuff, and use a `(declare is-handler?)`

or maybe it's time to split the dynamic application db stuff to its own ns?

Reply:
Yes, but not going to split it in this PR.